src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1111,7 +1111,6 @@ public static FunctionMetadata ReadFunctionMetadata(string scriptDir, Dictionary
                 string json = null;
                 try
                 {
-                    // This should be async
                     json = FileUtility.ReadAllText(functionConfigPath);
```

Comment:
Can the fileSystem you passed in be used here now?

Reply:
updated to use it if it's not null. Fallback to FileUtility if it's null.